README.md

```diff
@@ -46,15 +48,42 @@ jobs:
 
 With those steps complete, Robin AI will automatically run every time a pull request is opened or edited in your Github repository.
 
+### Gitlab
+
+To use Robin AI in your Gitlab project, you'll need to create `GITLAB_TOKEN` and `OPEN_AI_API_KEY` CI variables and create CI job. Here's how:
+
+1. In your Gitlab repository, open `Settings` -> `CI/CD` -> `Variables` and add `GITLAB_TOKEN` and `OPEN_AI_API_KEY` variables.
+2. In your repository, create or open `.gitlab-ci.yml` file.
+3. Copy and paste the following code into the file:
+
+```yml
+robin:ai:review:
+  image:
+    name: ghcr.io/Integral-Healthcare/robin-ai-reviewer
+    entrypoint: [""]
+  stage: review
+  script:
+    - >
+      ./entrypoint.sh \
+        --git_provider=gitlab \
+        --git_token=$GITLAB_TOKEN \
+        --open_ai_api_key=$OPEN_AI_API_KEY
+  rules:
+    - if: $CI_PIPELINE_SOURCE == 'merge_request_event'
+```
+
+_Note: make sure that you have `review` stage defined in your pipeline_
+
 ## Arguments
 
-| Name                | Required | Default Value             | Description                                                                                                       |
-|---------------------|----------|---------------------------|-------------------------------------------------------------------------------------------------------------------|
-| `GITHUB_TOKEN`      | Yes      | Automatically supplied    | A Github access token with the `repo` and `pull_request` scopes.                                                  |
-| `OPEN_AI_API_KEY`   | Yes      | N/A                       | An API key from Open AI's [developer portal](https://platform.openai.com/account/api-keys).                                                                       |
-| `gpt_model_name`    | No       | `gpt-3.5-turbo`           | The name of the GPT model to use for text generation.                                                             |
-| `github_api_url`    | No       | `https://api.github.com`  | The URL for the Github API endpoint. (Only relevant to enterprise customers.)                                      |
-| `files_to_ignore`   | No       |  (empty string)         | A whitespace delimited list of files to ignore.                                                                   |
+| Name | Required | Default Value | Description |
+|--|--|--|--|
+| `open_ai_api_key` | Yes | N/A | An API key from Open AI's [developer portal](https://platform.openai.com/account/api-keys). |
```

Comment:
Intention for the all caps `OPEN_AI_API_KEY` was to emphasize that it was a required argument

Reply:
Understood, I can change it here to have capital letters, but what about the usage from within the script? Should it also accept `OPEN_AI_API_KEY` as a parameter, or for consistency, stay as `open_ai_api_key`? It might be confusing if documentation is showing capitalized parameter while the entrypoint script accepts lower case argument.

Reply:
Whatever we elect for, we should be consistent. We should update the entry point script to accept an upper case argument in my opinion